opm/material/fluidsystems/blackoilpvt/GasPvtMultiplexer.hpp

```diff
@@ -116,6 +116,10 @@ class GasPvtMultiplexer
      */
     void initFromDeck(DeckConstPtr deck, EclipseStateConstPtr eclState)
     {
+        bool enableGas = deck->hasKeyword("GAS");
```

Comment:
(note that I essentially do not care about them, i.e., I will happily accept patches which use constant local integral variables.)